### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ishifoev**
#148

### CODE REVIEW

**Comment by Triantafullenia-Doumani** [comment]
If the User exists, instead of 

```
['opaque_id' => $userId]
```
The response should be in this form
```
["id"=>["idp"=>"some-idp","opaque_id"=>"some-opaque-user-id","type"=>1]]
```
So, the question is what values do we need to give to

* idp
* opaque_id

**Comment by ishifoev** [comment]
@Triantafullenia-Doumani I think, yeah it will be some hard coded for marie. We can take this response. You can check it again. But I think we need to see how it will be implement when you are send param with real data in furture.

**Review by Triantafullenia-Doumani — APPROVED** [review]

File: lib/Controller/RevaController.php [hunk 1]
```diff
@@ -711,6 +711,23 @@ public function Upload($userId, $path) {
...
 	}
+	/**
+	 * @PublicPage
+	 * @NoCSRFRequired
+	 * @NoSameSiteCookieRequired
+	 *
+	 * Get user list.
+	 */
+	public function GetUser($userId) {
+		$response = json_encode(["id" => ["idp" => "cesnet.cz","opaque_id" => "f7fbf8c8-139b-4376-b307-cf0a8c2d0d9c","type" => 1]]);
```
Comment (Triantafullenia-Doumani) [inline lib/Controller/RevaController.php:722]: You could remove json_encode()

Comment (ishifoev) [reply lib/Controller/RevaController.php]: Yeah done.

**Pull request merged**